src/main/java/br/ufrgs/inf/pet/dinoapi/entity/faq/Faq.java

```diff
@@ -0,0 +1,85 @@
+package br.ufrgs.inf.pet.dinoapi.entity.faq;
+
+import br.ufrgs.inf.pet.dinoapi.model.faq.FaqSaveRequestModel;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static br.ufrgs.inf.pet.dinoapi.constants.FaqConstants.TITLE_MAX;
+import static javax.persistence.GenerationType.SEQUENCE;
+
+@Entity
+@Table(name = "faq")
+public class Faq implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private static final String SEQUENCE_NAME = "faq_seq";
+
+    @Id
+    @GeneratedValue(strategy = SEQUENCE, generator = SEQUENCE_NAME)
+    @SequenceGenerator(name = SEQUENCE_NAME, sequenceName = SEQUENCE_NAME)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(name = "version", nullable = false)
+    private Long version;
+
+    @Column(name = "title", length = TITLE_MAX, nullable = false, unique = true)
+    private String title;
+
+    @OneToMany(mappedBy = "faq", cascade = CascadeType.ALL)
+    private List<FaqItem> items;
+
+    @OneToOne(mappedBy = "user")
+    private FaqUser faqUser;
```

Comment:
Uma faq não pode ter vários faqusers?

Reply:
não, uma faqUser é uma tabela terciária de relação 1x1 entre faq e user. uma faq teria uma relação one to many se a tabela faqUser não existisse